modules/install-roxctl-cli-linux.adoc

```diff
@@ -9,11 +9,12 @@ You can install the `roxctl` CLI binary on Linux by using the following procedur
 
 .Procedure
 
-. Download the latest version of the `roxctl` CLI:
+. Download the latest version of the `roxctl` CLI (supported architectures are `amd64`, `ppc64le` and `s390x`):
 +
 [source,terminal,subs=attributes+]
 ----
-$ curl -O https://mirror.openshift.com/pub/rhacs/assets/{rhacs-version}/bin/Linux/roxctl
+$ arch="$( (echo -n "-" && uname -m) | sed "s/-x86_64//")"
```

Comment:
For `ppc64le` and `s390x` Linux, `arch` will be `"-ppc64le"` and `"-s390x"` respectively.

For `x86_64` Linux, `arch` will be an empty string. This is intentional because this drives users to `/roxctl` artifacts where we did not denote architecture historically.
This can be changed to drive to `roxctl-amd64` but I don't think it's time for that yet.